website/server/controllers/api-v3/members.js

```diff
@@ -57,6 +57,288 @@ api.getMember = {
   },
 };
 
+function _addAchievement(achievementsResult, type, key, title, text, icon, category, value, earned) {
```

Comment:
All of these functions should take in an options object instead of positioned arguments. So when called, you would do:

```js
_addAchievement({
  achievementsResult: 'value', 
  type: 'something', 
  key: 'hey', 
  title: 'impressive thing', 
  text: 'owowow', 
  icon: 'hey', 
  category: 'yay', 
  value: 'foo',
  earned: true
});
```

Instead of 

```js
_addAchievement('value',  'something', 'hey', 'impressive thing', 'owowow', 'hey', 'yay',  'foo', true);
```

This has the benefit of providing named arguments, so you're never confused about what you're passing in and you don't have to worry about the position in which you pass it.